content/docs/guide/gamemodes/climb.md

```diff
@@ -11,16 +11,92 @@ tags:
+Whenever the player lands, they will be slowed for however much time is left on the timer.
+For reference, if a player jumps and lands on flat ground about 0.66 seconds or about half the time is taken.
+This means that jumping up to something slows you down longer since you land earlier. And jumping to something beneath will slow you less or not at all.  
+**The slowdown affects both horizontal and vertical movement: running is slower but also your jump height is reduced**.  
+**A Bhop, even a perfect one, does not circumvent the verticle movemnt penalty: Bhops will always get smaller height compared to a normal jump, thus they may not always give you the best distance or be able to reach high places**.  
+For speedrunning getting around this stamnia is a top prority for optimzing runs. For completion players only certain types of platforms make stamina a cause for concern namely ones that are to small or awkward to get a running start from a standsill.
+[This doc goes into futher detail](https://kzguide.gitlab.io/techniques/stamina/).
+
+---
+## Advanced Techniques 
+
+### Climb has some techniques of its own rarely seen outside of its gamemode.
+
+**Fast Ladder Climbing**: Ladders can be climbed by "walking into them". If you are directly facing the ladder hold W, if sideways A/D. If you face the ladder diagonally you can climb the ladder both ways. If you hold W and A/D the speeds add onto each other resulting in a much faster climb.    
+
+**Ladder Jumps**: The speed you gain from fast ladder climbing is enough to fling you into the air! This technique does not have anything to do with inputting jump (which just pushes you off the ladder) and is more simillar to surfing. Gain speed with the climb and flick off near the top. Just like surfing, the way you flick off dictactes your trajectory, and whether you trade speed for height and vice versa.  
+
+**Crouch Bhops**: Bhopping starting in a crouch position. It may be required in a map when going underneath a low celling. Doing a normal Bhop and crouching in the air will raise your collision box (you tuck in your legs), so to go underneath a hazard you must start the Bhop crounched and usually stay crouched.  ![image](../../../../assets/Crouch_Bhop.png)  
+Crouching will also limit your ablity to turn in the air and thus gain speed as well. This speed disadvantage is negated if you have already reached the Bhop speedcap on KZT since you will remain at 380 regardless of how much speed you try to gain.  
+
+**Standups**: While in the air, if you crouch close enough to the ground (TODO: get exact units) the engine will not let you uncrouch since you'll clip into the floor, instead it waits until the moment you've hit the ground or more space has opened up beneath you and uncrouches. This behavior gives you a nice grace period to execute a frame-perfect. (or tick-perfect) outcome. (Note that the faster you are falling, the shorter the grace period is).  
```

Comment:
should clarify that standups are only a thing in 1.6 kz.

> (TODO: get exact units)

18 units. standing collision height in goldsrc is 72, ducked is 36: half of 72. so when you duck the top of your collision box shrinks by half of 36 aka 18 units and the bottom shrinks by 18 units (this is a simplification of goldsrc collisions ofc, explaining goldsrc collisions is out of scope for this article)